data/hai/hai reveal 5 philosophers.txt

```diff
@@ -807,17 +815,87 @@ ship "Centipede" "Centipede (Teeneep Jump)"
 		"Outfits Expansion"
 		`"Biroo" Atomic Steering`
 		"Hai Diamond Regenerator"
-		"Hai Williwaw Cooling" 2
-		"Hai Chasm Batteries" 2
+		"Hai Corundum Regenerator"
+		"Hai Chasm Batteries"
+		"Liquid Nitrogen Cooler"
 		"Boulder Reactor"
 		Brig
-		"Fuel Pod" 3
+		"Fuel Pod" 2
+		Supercapacitor
```

Comment:
Did it perform badly?

Reply:
Looking at it I think it just had a rushed layout (had 10 outfit space remaining) and it would overheat **very** quickly
I think we could justify having some of these modifications (like extra fuel) done to the hull and making those more efficient, in another PR, but for now thats a very good change if you ask me (except if that 900 fuel was really needed but I dont think so)